### DISCUSSION TITLE
clarification

### DISCUSSION THREAD

**Discussion by atwahsz** [gempy-project/gempy#581] [Q&A] [answered]
spouse i have a csv file or a dataframe containing
X Y Z FACIES

how can i import ito gempy and does it pick up dimensions of the data (3d property) ?

also how can i visualize it in 3d and 2d cross sections

all of the best

Saleh.

**Comment by atwahsz** [comment]
also , how can i specify a color code

**Comment by javoha** [comment, marked as answer]
Hello @atwahsz,

based on your questions I would strongly adivce you to check the Gempy documentation (https://www.gempy.org/tutorials) and especially the tutorials. You will find explanations for your questions and a lot more there.
